components/esp_lvgl_port/test_apps/simd/functionality/main/Kconfig.projbuild

```diff
@@ -0,0 +1,5 @@
+# Creating CONFIG_LV_DRAW_SW_ASM_CUSTOM avaliable in lvgl Kconfig to enable assembler source files by deafult
+
+config LV_DRAW_SW_ASM_CUSTOM
```

Comment:
`LV_DRAW_SW_ASM_CUSTOM` is a config present in LVGL and I am not using neither `lvgl_port` , nor `lvgl` components, since the test app uses a hard copy of LVGL functions as their ANSI implementations. So there is no way to set this config, since it does not exist in the config. (I hope it works like that)

But, I need to set the `LV_DRAW_SW_ASM_CUSTOM` to true, so the assembly function prototypes present in the `esp_lvgl_port_lv_blend.h` are accessible for the test app.

```
#if !CONFIG_LV_DRAW_SW_ASM_CUSTOM
#warning "esp_lvgl_port_lv_blend.h included, but CONFIG_LV_DRAW_SW_ASM_CUSTOM not set. Assembly rendering not used"
#else
```

For this, I created this `projbuild` and set `LV_DRAW_SW_ASM_CUSTOM` to always true.

Reply:
Understood, thanks